modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/internal/util/StringUtil.java

```diff
@@ -91,6 +96,24 @@ public static boolean isBlank(String s) {
 		return false;
 	}
 
+	public static boolean isUrl(String url) {
+		if (!isBlank(url)) {
```

Comment:
Can you flatten the nesting here by using an early return for `if isBlank`?

Reply:
Sure, will do. Thanks! :)